LoopKit/InsulinKit/OutsideDoseEvent.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  OutsideDoseEvent.swift
+//  LoopKit
+//
+//  Created by Anna Quinlan on 4/17/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+import Foundation
+import CoreData
+
+class OutsideDoseEvent: PumpEvent {
```

Comment:
An outside dose event contains basically all the info of a `PumpEvent`, but makes it so we can easily distinguish between a logged dose and a pump-delivered dose